.github/workflows/enos-run.yml

```diff
@@ -129,11 +129,13 @@ jobs:
         uses: hashicorp/action-setup-enos@v1    # TSCCR: loading action configs: failed to query HEAD reference: failed to get advertised references: authorization failed
         with:
           github-token: ${{ secrets.SERVICE_USER_GITHUB_TOKEN }}
-      - name: Set up AWS SSH private key
+      - name: Prepare scenario dependencies
+        id: prepare_scenario
         run: |
           mkdir -p ./enos/support
           echo "${{ secrets.SSH_KEY_PRIVATE_CI }}" > ./enos/support/private_key.pem
           chmod 600 ./enos/support/private_key.pem
+          echo "debug_data_artifact_name=enos-debug-data_$(echo ${{ matrix.scenario }} | sed -e 's/ /_/g' | sed -e 's/:/=/g')" >> $GITHUB_OUTPUT
```

Comment:
There are some linter errors here
```
.github/workflows/enos-run.yml:134:9: shellcheck reported issue in this script: SC2086:info:4:121: Double quote to prevent globbing and word splitting [shellcheck]
    |
134 |         run: |
    |         ^~~~
.github/workflows/enos-run.yml:134:211: property "scenario" is not defined in object type {filter: string} [expression]
```

Reply:
Fixed. The linter actually caught one real issue... as I had copied this from the vault repo, where they use `matrix.scenario` rather than `matrix.filter` for the enos filter.